src/casm/app/enum/methods/ConfigEnumSiteDoFsInterface.cc

```diff
@@ -347,7 +353,7 @@ void parse(InputParser<ConfigEnumSiteDoFsParams> &parser,
   // "max_nonzero" -> params.max_nonzero
   // note that help indicates default==axes.rows(), but that is
   // params.axes.cols()
-  parser.optional_else(params.min_nonzero, "max_nonzero",
+  parser.optional_else(params.max_nonzero, "max_nonzero",
                        Index{params.axes.cols()});
```

Comment:
I suppose this was a typo in the code. Hence I changed it. Could you verify this @bpuchala? 
min_nonzero was being populated as max_nonzero?